webapp/src/css/enketo/medic.less

```diff
@@ -21,16 +21,13 @@
 
     .dropdown-menu .option-wrapper {
       padding: 3px;
-      .option-label {
-        width: calc(100% - 34px);
-      }
     }
 
     .question:not(.or-appearance-likert):not(.or-appearance-columns) .option-wrapper {
       .option-label.active {
         white-space: normal;
         word-break: break-word;
-        display: contents;
+        display: inline;
```

Comment:
I'm not proposing this change, this is a placeholder workaround for https://github.com/medic/cht-core/issues/8669